src/commands/price.ts

```diff
@@ -1,38 +1,24 @@
-import Alpha from 'alphavantage';
 import { Config } from '../config';
 
 const { SlashCommandBuilder } = require('@discordjs/builders');
 
 export = {
-  _alpha: undefined,
-
   data: new SlashCommandBuilder()
     .setName('price')
     .setDescription('Get price of a ticker')
     .addStringOption((option) => option.setName('ticker').setDescription('The ticker to get a quote')),
 
   async execute(interaction, config: Config) {
-    if (typeof this.alpha === 'undefined') {
-      this.alpha = Alpha({ key: config.alphaKey });
-    }
-    const { alpha } = this;
-
```

Comment:
@rmbreak i'm running into issues importing this globally, I also don't understand this code you added previously. I tried something similar and it didn't work for me. Thoughts?

Reply:
> i'm running into issues importing this globally, I also don't understand this code you added previously.

What issues? Error message? Behavior?

> I also don't understand this code you added previously. I tried something similar and it didn't work for me. Thoughts?

Is there something specific in this code block that you don't understand? It's lazily initializing `Alpha` on the first use (first call to `execute`) and caches `Alpha` within `this.alpha`. What issues were you running into?
```ts
  async execute(interaction, config: Config) {
    if (typeof this.alpha === 'undefined') {
    // eslint-disable-next-line global-require
      this.alpha = Alpha({ key: config.alphaKey });
    }
    const { alpha } = this;
    ...
}
```